oap-server/server-storage-plugin/storage-banyandb-plugin/src/main/java/org/apache/skywalking/oap/server/storage/plugin/banyandb/stream/BanyanDBEBPFProfilingTaskDAO.java

```diff
@@ -108,24 +109,15 @@ protected void apply(StreamQuery query) {
     }
 
     @Override
-    public EBPFProfilingTask queryById(String id) throws IOException {
+    public List<EBPFProfilingTaskRecord> queryByLogicalId(String id) throws IOException {
```

Comment:
What does logic ID mean?

Reply:
logical ID is used on the network profiling, we need one ID to keep the profiling task alive if needed.

Reply:
I mean, is it task ID or something? From a naming perspective, logic ID looks like a general name, I want it relative with profiling task.

Reply:
Oh, it's task ID. I just let the DAO return the original task storage data, not the query data. Then, the service decides how to use the stored data(reduce the logical code in the DAO).

Reply:
OK, then rename this to `getTaskRecord`?

Reply:
Ok, renamed.